script/Arcueid/Day 1/01_00_ARC01_1_2.txt

```diff
@@ -94,8 +94,8 @@ Senpai leans in close to look me in the eye.
 [sha:c954dad85579d23ac1b90486a6fff49e6923d9b4]{
 -- Page 52, Offset 1284.
 -- よろしい。それでしたら放課後、いつもの茶道部に来てください。貴方なら大歓迎です」
-Alright. In that case, come by the tea ceremony club after school.
- I'll be more than happy to have you."
+Very well. In that case, come by the tea ceremony club after school.
+ You're more than welcome."
```

Comment:
I prefer the original for the third sentence, but no real reasoning. I just like the way it sounds better.